extension/js/common/lang.ts

```diff
@@ -89,11 +93,11 @@ export const Lang = { // tslint:disable-line:variable-name
   general: {
     contactMinimalSubsentence,
     contactIfHappensAgain,
-    contactIfNeedAssistance: (isFesUsed: boolean) => isFesUsed ? 'Contact your Help Desk if you need assistance.' : 'Email human@flowcrypt.com if you need assistance.',
+    contactIfNeedAssistance: (isFesUsed?: boolean) => contactForSupportSubsentence(isFesUsed || isEnterpriseBuildUsed, 'if you need an assistance.'),
     somethingWentWrongTryAgain: (isFesUsed: boolean) => `Something went wrong, please try again. ${contactIfHappensAgain(isFesUsed)}`,
     contactForSupportSubsentence,
-    contactForSupportSentence: (isFesUsed: boolean) => isFesUsed ? 'Please contact your Help Desk for support.' : 'Please write us at human@flowcrypt.com to fix it.',
-    writeMeToFixIt: (isFesUsed: boolean) => isFesUsed ? 'Please contact your Help Desk for support.' : 'Please write us at human@flowcrypt.com to get this resolved.',
+    contactForSupportSentence: (isFesUsed?: boolean) => contactForSupportSubsentence(isFesUsed || isEnterpriseBuildUsed, 'for support.'),
+    writeMeToFixIt: (isFesUsed: boolean) => contactForSupportSubsentence(isFesUsed, 'to fix it.'),
     restartBrowserAndTryAgain: (isFesUsed: boolean) => `Unexpected error occured. Please restart your browser and try again. If this persists after a restart, ${contactForSupportSubsentence(isFesUsed)}.`,
```

Comment:
Then here you wouldn't need the `isEnterpriseBuildUsed` at all, if it's already implemented in the choke points. That would mean less repetition.

Reply:
thank you, sir. I think I got the main idea.